src/Files.App/Actions/FileSystem/AddItemAction.cs

```diff
@@ -24,7 +24,7 @@ internal class AddItemAction : ObservableObject, IAction
 
 		public RichGlyph Glyph { get; } = new(opacityStyle: "ColorIconNew");
 
-		public bool IsExecutable => context.CanCreateItem && !context.HasSelection;
```

Comment:
This change affects more than just column layout. Are there any regression concerns with removing this? I don't know why the action is disabled while items are selected though.

Reply:
That's right, it also fixes an issue where the option was disabled when items were selected.

Reply:
@ferrariofilippo why did you add `!context.HasSelection` to `IsExecutable`?
https://github.com/files-community/Files/commit/f52fda7b09b2f197431e406150e7571fb3fa7697

Reply:
I assume it was to hide the option when right clicking on multiple files. However, this is handled in the context menu helper so it's okay to remove it from the action.